shell/packages/accounts-email-token/token_templates.html

```diff
@@ -0,0 +1,3 @@
+<template name="_emailTokenError">
+  <h2>Failed to login with token. It may be expired or you may have copied it in wrong.</h2>
```

Comment:
"Invalid authentication code. If it has been more than 15 minutes, the token has expired, otherwise you may have simply mistyped it."